bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/WemoHandler.java

```diff
@@ -195,19 +227,23 @@ public void onValueReceived(@Nullable String variable, @Nullable String value, @
 
         updateStatus(ThingStatus.ONLINE);
 
+        if ("EnergyPerUnitCost".equals(variable)) {
```

Comment:
You need to include handled variables instead of excluding known unhandled ones. Otherwise **NewUnknownVariable** would be added and could risk breaking parser at worst, or consume a small piece of memory unneededly at best.

Reply:
Done